sklearn/utils/multiclass.py

```diff
@@ -344,11 +344,33 @@ def _check_partial_fit_first_call(clf, classes=None):
 
         else:
             # This is the first call to partial_fit
+            if issparse(y):
+                raise ValueError("Sparse y is not supported.")
+
+            # Validate classes against unique labels in y
```

Comment:
Since this test ( of checking for mismatch between classes and unique y ) is brought to a more general test, wouldn't it be appropriate to support multi-output ? ( especially since there is interest in adding `partial_fit` capabilities to more estimators... )